extensions/mssql/src/objectManagement/ui/restoreDatabaseDialog.ts

```diff
@@ -235,7 +235,6 @@ export class RestoreDatabaseDialog extends ObjectManagementDialogBase<Database,
 		// Restore from
 		const restoreFromDropdownOptions = this.isManagedInstance ? [localizedConstants.RestoreFromUrlText, localizedConstants.RestoreFromS3UrlText] : [localizedConstants.RestoreFromDatabaseOptionText, localizedConstants.RestoreFromBackupFileOptionText, localizedConstants.RestoreFromUrlText, localizedConstants.RestoreFromS3UrlText];
 		this.restoreFrom = this.createDropdown(localizedConstants.RestoreFromText, async (newValue) => {
-			this.backupFilePath = this.backupURLPath = '';
```

Comment:
Other small fix: this line was causing the file input box to be cleared when switching modes back and forth. These cached values get used immediately below this to restore the input box value, so they don't need to be here.